modules/aviatrix-controller-build/variables.tf

```diff
@@ -110,6 +110,12 @@ variable "controller_name" {
   default     = ""
 }
 
+variable "private_mode" {
+  type        = string
+  description = "Enable private mode"
+  default     = false
+}
+
```

Comment:
We also need to make it clear in DOC that in private mode, user has to deploy the VPC on their own. Since in this module we're not deploying NAT gateway, subnet won't have internet access.